### PR TITLE
Fix region exploring for Julich brain 2.4

### PR DESCRIPTION
**Pull request by fsdavid**

### CODE REVIEW

File: src/ui/parcellationRegion/region.base.spec.ts [hunk 1]
```diff
@@ -49,7 +51,8 @@ const mr1rh = {
 
 const mr0lh = {
   labelIndex: 1,
-  name: 'mr0 - left hemisphere',
+  name: 'mr0',
+  status: 'left hemisphere',
   fullId: {
     kg: {
       kgSchema: 'fzj/mock/pr',
```
Comment (xgui3783) [inline src/ui/parcellationRegion/region.base.spec.ts:58]: this is actually not what I meant.

the existing tests needs to stay,

but the logic needs to accomodate both the `region name - hemisphere` and `name` `status` scenario

**Review by xgui3783 — CHANGES_REQUESTED** [review]

**Comment by xgui3783** [comment]
I would say, just leave the PR, I can take a look at it. I think I can fix this issue.

More important is, if you can take a look at connectivity components, and see if there are any bugs

File: src/ui/parcellationRegion/region.base.spec.ts (hunk 1 above)
Comment (fsdavid) [reply src/ui/parcellationRegion/region.base.spec.ts:58]: actually yes, the new approach was only for working for Julich brain 2.4 I will fix it for the old version too

**Comment by fsdavid** [comment]
> I would say, just leave the PR, I can take a look at it. I think I can fix this issue.
> 
> More important is, if you can take a look at connectivity components, and see if there are any bugs

I can look at both. I will fix this issue and I can already found a bug in the connectivity browser.

**Comment by fsdavid** [comment]
Now both versions of julich brain should work

**Comment by xgui3783** [comment]
Thanks. 

I will add unit tests then merge

**Pull request merged**